libs/search/src/lib/search/search.component.html

```diff
@@ -1,9 +1,7 @@
 <p>Search:</p>
 
 <ng-workspace05-nx-input></ng-workspace05-nx-input>
-<label>
-  <input type="checkbox" [(ngModel)]="isGlobalSearch" /> Global search?
-</label>
+<ng-workspace05-nx-checkbox [checked]="true" (checkedChange)="isGlobalSearch = $event">Global search?</ng-workspace05-nx-checkbox>
```

Comment:
This is not correct state binding, you should simply use 2-way binding to `checked` input:
```suggestion
<ng-workspace05-nx-checkbox [(checked)]="isGlobalSearch">Global search?</ng-workspace05-nx-checkbox>
```